extra_views_tests/models.py

```diff
@@ -39,6 +40,15 @@ def __unicode__(self):
         return '%s (%s)' % (self.name, self.sku)
 
 
+class Contact(models.Model):
+    name = models.CharField(max_length=255)
+    email = models.CharField(max_length=255)
+    order = models.ForeignKey(Order, related_name='contacts', on_delete=models.CASCADE)
+
+    def __unicode__(self):
```

Comment:
__str__(self): should be better as python 3 is the currently supported version of django.